.github/ISSUE_TEMPLATE/1_3_native_image_crash_report.yml

```diff
@@ -0,0 +1,59 @@
+name: 💥 Share Native Image Crash Report (svm_err_b_*.md)
```

Comment:
Try `svm_err_b_*.md` if it looks nicer than svm_err_b_*.md.

Reply:
Share Native Image Build-Time Crash Report

Reply:
Done